R/dot_compare_to_field.R

```diff
@@ -20,7 +20,7 @@
 	obs_metric <- vector('numeric', length = length(un_dates))
 	dates <- as.POSIXct(rep(NA, length = length(un_dates)))
 	
-	for (j in 1:length(un_dates)){
+	for (j in seq_len(length(un_dates))){
```

Comment:
Nice. I should use this more. It solves the 1:0 problem I've run into in the past.